packages/react/src/components/Loader/index.tsx

```diff
@@ -1,9 +1,17 @@
 import React from "react";
+import classNames from "classnames";
 
 interface Props {
   "aria-label"?: string;
+  size?: "small" | "medium";
```

Comment:
In facts `medium` is the default size but indeed it does not appears in the CSS file because it is the default size, but anyway I will rework the CSS to make it appear